src/korean.js

```diff
@@ -40,108 +72,68 @@ class Korean {
-      let brokeWord = breakdown(word[wordLength - 2]);
-      let brokeLength = brokeWord.length;
-      let syllableEnd = brokeWord[brokeLength - 1];
-      let stemWord = word.slice(0, wordLength - 2);
+      const brokeWord = breakdown(word[wordLength - 2]);
+      const brokeLength = brokeWord.length;
+      const syllableEnd = brokeWord[brokeLength - 1];
+      const stemWord = word.slice(0, wordLength - 2);
       let newSyllable = brokeWord.slice(0, brokeLength - 1);
 
-      switch(syllableEnd) {
+      switch (syllableEnd) {
         case 0:
         case 1:
         case 4:
           // if last letter is ㅏ leave alone
           return word.slice(0, wordLength - 1);
         case 8:
           // replace with: ㅘ (9)
-            // concat back to word
-            newSyllable.push(9);
-            newSyllable = combineSymbols(newSyllable);
-            return stemWord + newSyllable;
+          // concat back to word
+          newSyllable.push(9);
+          newSyllable = combineSymbols(newSyllable);
+          return stemWord + newSyllable;
         case 18:
-        // vowel ㅡ replace with ㅓ (4)
-              newSyllable.push(4);
-              newSyllable = combineSymbols(newSyllable);
-            if(wordLength <= 2) {
-              return newSyllable;
-            } else {
-              return stemWord + newSyllable;
-            }
-       case 13:
-        // 13 (ㅜ) convert to 14 (ㅝ)
+          // vowel ㅡ replace with ㅓ (4)
+          newSyllable.push(4);
+          newSyllable = combineSymbols(newSyllable);
+          if (wordLength <= 2) {
+            return newSyllable;
+          }
+          return stemWord + newSyllable;
+        case 13:
+          // 13 (ㅜ) convert to 14 (ㅝ)
           newSyllable.push(14);
           newSyllable = combineSymbols(newSyllable);
-          if(wordLength <= 2) {
+          if (wordLength <= 2) {
             return newSyllable;
-          } else {
-            return stemWord + newSyllable;
           }
-       case 20:
-        // vowel ㅣ replace with ㅕ(6)
-          newSyllable.push(6)
+          return stemWord + newSyllable;
+        case 20:
+          // vowel ㅣ replace with ㅕ(6)
+          newSyllable.push(6);
           newSyllable = combineSymbols(newSyllable);
-          if(wordLength <= 2) {
+          if (wordLength <= 2) {
             return newSyllable;
-          } else {
-            return stemWord + newSyllable;
           }
+          return stemWord + newSyllable;
         default:
-            // if consonant
-            conjugate = word.slice(0, wordLength - 1);
-            return conjugate.concat('어');
+          // if consonant
+          conjugate = word.slice(0, wordLength - 1);
+          return conjugate.concat('어');
       }
     } // end of first else
   } // end of presentWord function
 
-  doPast(word) {
-    let presentWord = doPresent(word);
+  doPast (word) {
+    console.log(word);
```

Comment:
@cecilebertin @yjlim5 I have a pull request for reference to Heggy here https://github.com/minsooshin/conjugator/pull/1. Actually, I don't understand why we need to have `presentWord` at the top of `doPast` method. It transfers `하다` to `해` first, but we don't need that for past because `했다` doesn't cover all the past cases. If we transfer to `하다` to `해` first, there will be more job for other cases.